src/main/java/com/google/sps/servlets/YoutubeServlet.java

```diff
@@ -0,0 +1,25 @@
+// Copyright 2019 Google LLC
```

Comment:
We may want to update the license to 2020 :) (There was a discussion on intern host chat about this, it seems better to have it updated, even though it is pretty trivial)
Below as well.